typescript/rest-express-auth/prisma/seed.ts

```diff
@@ -0,0 +1,70 @@
+import { PrismaClient, Prisma } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+const userData: Prisma.UserCreateInput[] = [
+  {
+    email: 'alice@prisma.io',
+    password: 'testPassword',
```

Comment:
I would recommend the following for the test data in the event a user tries to log in with the test user:
```suggestion
    password: await argon2.hash('testPassword'),
```